frontend/src/components/RFI/RFILanding.vue

```diff
@@ -852,152 +799,63 @@
+                  @click="removeObjFromList(index, model.expansionList)"
+                >
+                  mdi-close
+                </v-icon>
+
                 <v-col class="col-md-1 col-12 ml-md-n8">
-                  <v-menu
-                    ref="menufrom"
-                    v-model="obj.menufrom"
-                    v-model:return-value="timefrom"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field
-                        v-model="obj.timefrom"
-                        :disabled="isReadOnly"
-                        label="From"
-                        readonly
-                        v-bind="props"
-                      />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.menufrom"
-                      v-model="obj.timefrom"
-                      :disabled="isReadOnly"
-                      full-width
-                      @click:minute="$refs.menufrom[index].save(timefrom)"
-                    />
-                  </v-menu>
-                </v-col>
+                  <AppTimeInput
+                    v-model="obj.timefrom"
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="open"
+                  />
 
-                <v-col class="col-md-1 col-12">
-                  <v-menu
-                    ref="menuto"
-                    v-model="obj.menuto"
-                    v-model:return-value="timeto"
+                  <!-- we need logic to prevent choosing a time before -->
+
+                  <AppTimeInput
+                    v-model="obj.timeto"
                     :disabled="isReadOnly"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field v-model="obj.timeto" :disabled="isReadOnly" label="To" readonly v-bind="props" />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.menuto"
-                      v-model="obj.timeto"
-                      :disabled="isReadOnly"
-                      :min="obj.timefrom"
-                      full-width
-                      @click:minute="$refs.menuto[index].save(timeto)"
-                    />
-                  </v-menu>
+                    full-width
+                    max-width="200px"
+                    label="close"
```

Comment:
I think the original label was 'To'. Do we have a requirement for changing it to 'close'?

Reply:
good point - i've changed these all